content/verification/address-validation/release-notes/2024-05-15.md

```diff
@@ -0,0 +1,10 @@
+Released version 3 of address validation. Updates include:
+- `/validate` and `/candidate` endpoints
+
+     - Removed support for US Congressional District. 
```

Comment:
Should this be a sub-bullet  of 'validate' and 'candidate above? Or its own bullet point?
```suggestion
     - Removing support for US Congressional District. 
```